docs/api/loaders/ImageBitmapLoader.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<base href="../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		<h1>[name]</h1>
+
+		<div class="desc">
+			A loader for loading an [page:Image] as an [link:https://developer.mozilla.org/de/docs/Web/API/ImageBitmap ImageBitmap].
```

Comment:
Perhaps add a note about why someone might want to use this, e.g.:

> An `ImageBitmap` provides an asynchronous and resource efficient pathway to prepare textures for rendering in WebGL.

Reply:
@WestLangley I remember we said some time ago that the documentation should not focus on tutorial stuff. Therefore i struggle a little but with adding an explanation of `ImageBitmap`. What do you think?

Reply:
I think the short explanation Don suggests is good though. One can go to MDN after that is needed.

This is pretty much (sadly) Chrome only at the moment anyway 🙁

Reply:
Ok by me with or without the explanation. ImageBitmap may also be too new to justify writing best practices for its use — still not working in Firefox or Safari Tech Preview for me here, despite partial implementation. Down the road I see it as a recommended way to load images into an already-running scene, but not necessarily better for up-front initialization.

Reply:
@Mugen87 I agree with @mrdoob here. I know this is not a black-and-white issue.

Reply:
Ok, i've added the suggestion of @donmccurdy 👍